tasks/util/openmpi.py

```diff
@@ -11,7 +11,7 @@
 NATIVE_HOSTFILE = "/home/mpirun/hostfile"
 
 HOSTFILE_LOCAL_FILE = "/tmp/hostfile"
-SLOTS_PER_HOST = 2  # This must match the number of vCPUs in the VM used
+SLOTS_PER_HOST = 4  # This must match the number of vCPUs in the VM used
```

Comment:
I don't think we normally use inline comments like this although it's not enforced by black. 

Instead I'd have something like:

```python 
# NOTE: the slots per host must be the same as the number of vCPUs
# in the VMs in the k8s deployment. The VM type is defined at:
# <url to github>
SLOTS_PER_HOST = 4
```